orchagent/mirrororch.cpp

```diff
@@ -260,6 +265,76 @@ bool MirrorOrch::decreaseRefCount(const string& name)
     return true;
 }
 
+bool MirrorOrch::validateDstPort(const string& dstPort)
+{
+    Port port;
+    if (!m_portsOrch->getPort(dstPort, port) && port.m_type != Port::PHY)
+    {
+        SWSS_LOG_ERROR("Failed to locate port %s", dstPort.c_str());
+        return false;
+    }
+    return true;
+}
+
+bool MirrorOrch::checkPortExistsInSrcPortList(const string& port, const string& srcPort)
+{
+    auto ports = tokenize(srcPort, ',');
+    if (ports.size() != 0)
+    {
+        for (auto alias : ports)
+        {
+            if(port == alias)
+            {
+                return true;
+            }
+        }
+    }
+
+    return false;
+}
+
+bool MirrorOrch::validateSrcPort(const string& srcPort)
+{
+    auto ports = tokenize(srcPort, ',');
+
```

Comment:
Delete this line, to be consistent with above function

Reply:
This is source port list, so modified function name to reflect it.